qt/python/mantidqt/mantidqt/widgets/plotconfigdialog/curvestabwidget/presenter.py

```diff
@@ -125,7 +124,7 @@ def _replot_current_curve(self, plot_kwargs):
             curve_index = ax.get_lines().index(curve)
             errorbar = False
         else:
-            curve_index = [artist for artist in ax.containers if isinstance(artist, ErrorbarContainer)].index(curve)
+            curve_index = ax.get_lines().index(curve[0])
```

Comment:
Should we put a type check in to make sure `curve` is an `ErrorbarContainer`? Maybe if `curve` is neither a `Line2D` or an `ErrorbarContainer` it could raise an exception?